src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -348,6 +349,10 @@ public function askQuestion(Question $question)
 
         if (!$this->questionHelper) {
             $this->questionHelper = new SymfonyQuestionHelper();
+
+            if ($this->inputStream) {
```

Comment:
Good catch @sstok, but I think it needs to keep an additional check in ask() because in most cases the helper will not be set when calling setInputStream, because it is set in ask()